lua/obsidian/img_paste.lua

```diff
@@ -87,9 +87,10 @@ end
 ---@param fname string|?
 ---@param default_dir obsidian.Path|string
 ---@param default_name string|?
+---@param should_confirm boolean|?
```

Comment:
All 4 `@param` annotations would be condensed to this:
```suggestion
---@param opts { fname: string|?, default_dir: obsidian.Path|string|?, default_name: string|?, should_confirm: boolean|? }|?
```

Reply:
Thanks! I've made these changes!